### PR TITLE
Improve log strings and never log to stdout.

### PR DESCRIPTION
**Pull request by SimonTeixidor**
Remove log message to `stdout`, and stop using debug formatting for nicer looking logs.

### CODE REVIEW

File: src/session.rs [hunk 1]
```diff
@@ -213,7 +213,7 @@ impl Session {
             0xad => {
                 let msg: protocol::keyexchange::APLoginFailed =
                     protobuf::parse_from_bytes(&data).unwrap();
-                error!("Authentication failed, {:?}", msg);
+                error!("Authentication failed, {}", msg);
```
Comment (plietar) [inline src/session.rs]: `msg` doesn't implement `Display` (cf Travis logs)

File: src/player.rs [hunk 2]
```diff
@@ -180,12 +180,12 @@ fn find_available_alternative<'a>(session: &Session, track: &'a Track) -> Option
 fn load_track(session: &Session, track_id: SpotifyId) -> Option<vorbis::Decoder<Subfile<AudioDecrypt<Box<ReadSeek>>>>> {
     let track = session.metadata::<Track>(track_id).await().unwrap();
 
-    info!("Loading track {:?}", track.name);
+    info!("Loading track {}", track.name);
```
Comment (plietar) [inline src/player.rs]: Keeping quotes around the name would be good.
That's the reason I was using debug formatting, although I agree it's not great for non ASCII names.

**Review by plietar — CHANGES_REQUESTED** [review]

File: src/session.rs (hunk 1 above)
Comment (SimonTeixidor) [reply src/session.rs]: Well shit, I don't know how that one ended up in the commit. Will fix.

**Comment by SimonTeixidor** [comment]
I agree with quoting the song names. I pushed a fix.

**Comment by plietar** [comment]
Cool, thanks

**Pull request merged**